src/patternfly/components/Nav/nav.scss

```diff
@@ -323,40 +222,30 @@
   // List simple link
   .pf-c-nav__link {
     display: block;
-    padding:
-      var(--pf-c-nav__simple-list-link--PaddingTop)
-      var(--pf-c-nav__simple-list-link--PaddingRight)
-      var(--pf-c-nav__simple-list-link--PaddingBottom)
-      var(--pf-c-nav__simple-list-link--PaddingLeft);
+    padding: var(--pf-c-nav__simple-list-link--PaddingY) var(--pf-c-nav__simple-list-link--PaddingX);
     font-weight: var(--pf-c-nav__simple-list-link--FontWeight);
     color: var(--pf-c-nav__simple-list-link--Color);
     background-color: var(--pf-c-nav__simple-list-link--BackgroundColor);
 
     // States
     &.pf-m-hover,
-    &:hover {
-      color: var(--pf-c-nav__simple-list-link--active--Color);
-      background-color: var(--pf-c-nav__simple-list-link--active--BackgroundColor);
-    }
-
+    &:hover,
     &.pf-m-focus,
-    &:focus {
-      font-weight: var(--pf-c-nav__simple-list-link--focus--FontWeight);
-      color: var(--pf-c-nav__simple-list-link--focus--Color);
-      background-color: var(--pf-c-nav__simple-list-link--focus--BackgroundColor);
-    }
-
+    &:focus,
     &.pf-m-active,
-    &:active {
-      font-weight: var(--pf-c-nav__simple-list-link--active--FontWeight);
-      color: var(--pf-c-nav__simple-list-link--active--Color);
-      background-color: var(--pf-c-nav__simple-list-link--active--BackgroundColor);
+    &:active,
+    &.pf-m-current {
+      --pf-c-nav__simple-list-link--Color: var(--pf-c-nav__simple-list-link--hover--Color);
+
+      background-color: var(--pf-c-nav__simple-list-link--hover--BackgroundColor);
```

Comment:
can you redefine `--pf-c-nav__simple-list-link--BackgroundColor` here?

Reply:
Hmm, I removed this line and added `--pf-c-nav__list-link--BackgroundColor: var(--pf-c-nav__simple-list-link--hover--BackgroundColor);` and it worked?